pkg/aggregator/ckey/tests/collisions_test.go

```diff
@@ -0,0 +1,42 @@
+package ckey_test
+
+import (
+	"fmt"
+	"io/ioutil"
+	"strings"
+	"testing"
+
+	"github.com/DataDog/datadog-agent/pkg/aggregator/ckey"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestCollisions(t *testing.T) {
+	assert := assert.New(t)
+
+	data, err := ioutil.ReadFile("./random_sorted_uniq_contexts.csv")
```

Comment:
should this `t.Skip()` if the file does not exist?

Reply:
I would say that it's not hurting to have one small sample shipped to have the test running every pipeline run in order to force the maintenance of the test itself, since it may not be used manually very often. WDYT?

Reply:
Ah, I was thinking that this wouldn't run by default -- it seems a lot of CPU and repository size to sacrifice to make a probabilistic assertion about a hash function that's implemented in a different package.  In other words, I'm not sure what this test would ever catch by running in CI (especially using the same input corpus on every run, making it deterministic).

But I'm happy to be told otherwise!